packages/trader/src/Modules/MT5/Containers/mt5-compare-accounts-modal.jsx

```diff
@@ -112,14 +112,12 @@ const compareAccountsData = ({ landing_companies }) =>
                 <MT5AttributeDescriber
                     name={localize('Commission')}
                     counter={4}
-                    tooltip={localize(
-                        'Deriv charges no commission across all account types, except cryptocurrency accounts.'
-                    )}
+                    tooltip={localize('Deriv charges no commission across all account types, except cryptocurrencies.')}
```

Comment:
can you change it to `except for cryptocurrencies`? let's match the grammar of binary.com